### ISSUE TITLE
Dynamic Type/accessible font sizes

### ISSUE THREAD

**Issue by zmknox** [DagAgren/toot#99]
As someone with low vision, I turn up the iOS system dynamic type font size setting. It would be really great if timelines, notifications, and other text in the app properly scaled based on the system dynamic type setting.

**Comment by billsaysthis** [comment]
This is crucial for me as well, current text is really hard for me to read.

**Comment by Spudboy** [comment]
I agree with billsaysthis & zmknox current font size on iPhone is unusable by me.

**Comment by frankgerhardt** [comment]
On an iPad the small fixed font size is a pain.

**Comment by contraexemplo** [comment]
Came here to give this a +1. I’m also a low vision user who finds it difficult to use the app since it doesn’t offer any text size configuration nor follows iOS accessibility settings. It’s the only thing that is preventing me from using this app as much as I wanted.

**Comment by ludwick** [comment]
I feel kind of foolish because I bought the app on the store and didn't check that this was supported first. I can't really use the app at all as it is. :( 

I read thru the app description before buying and didn't see anything about it NOT supporting either system text size or having a way to adjust. Could you add that at least so future folks don't do what I did?

**Comment by cprosser** [comment]
Just bought the app. Love the approach, but barely usable by me since it doesn’t follow the increased system font size I have.

**Comment by rbruchal** [comment]
+1 for me as well. Even if you don't have a specific visual disability, eventually you'll probably want to increase the font size. Time catches up to all of us, lol.

**Comment by freediverx** [comment]
My vision is generally fine, to the point where I don’t wear glasses for most activities. But as anyone over 40 will tell you, your close distance vision will eventually deteriorate, and I’d prefer not to have to wear reading glasses to use this app.

**Comment by bhhawthorne** [comment]
This is the only thing keeping me from using the app regularly. Please support Dynamic Text.

**Comment by fivetonsflax** [comment]
+1 Wins Six Beers If Fixed

**Comment by alexbbrown** [comment]
+1 for me.  Since I was 45 I have had to turn font sizes up 10% every year in the standard apple apps.

I can't comfortably use your app without reading glasses, and I don't carry reading glasses for walking around, which means I'll have to use another app unless I'm sitting down at home.

This issue will affect most adults at some point in their lifetime.
